### DISCUSSION TITLE
iOS Mail - Move to Junk (and train Spamassassin)

### DISCUSSION THREAD

**Discussion by mr-smit** [docker-mailserver/docker-mailserver#4291] [General]
Is there a way to train Spamassassin after I move email to Junk folder in iOS Mail app ?

Because right now if I mark it as Spam by moving to Junk folder in iOS Mail app, when the senders sends me again similar email, it is not marked as spam.

**Comment by audioscavenger** [comment]
spamassasin is supposed to retrain itself by watching the junk folder, that's how poste.io and mailu are setup. Unless I am wrong, this is not setup properly here, yet.
